torch/_refs/__init__.py

```diff
@@ -1366,6 +1366,11 @@ def fmod(a: TensorLikeType, b: TensorLikeType) -> TensorLikeType:
     return prims.fmod(a, b)
 
 
+@register_decomposition(aten.frexp)
+def frexp(self: TensorLikeType) -> Tuple[TensorLikeType, TensorLikeType]:
```

Comment:
This should have a `PythonRefInfo` in `torch/testing/_internal/common_methods_invocations.py`. Also, `frexp` does have an out overload so `@out_wrapper` would be appropriate.

Reply:
For the out wrapper, I'm not sure how to do that. `torch.frexp` has a `out` parameter which takes in a tuple of Tensors and `aten.frexp` has two out parameters `mantissa, exponent`. I'm not sure how to reconcile this difference.
Setting `@out_wrapper("mantissa", "exponent")` fails in an error.

The other methods that have different names like `std_mean, var_mean` have `supports_out=False` in their opinfos.

Reply:
Thanks. That fixed the issues for me.